mslib/msui/mscolab.py

```diff
@@ -1788,7 +1788,7 @@ def archive_operation(self):
                     "token": self.token,
                     "op_id": self.active_op_id,
                     # when a user archives an operation we set the max “natural” integer in days
-                    "days": sys.maxsize,
+                    "days": 99999,  # larger values run into a lot of issues
```

Comment:
Unfortunately someone with an old version will set the value where you see a lot of issues. This can be solved when we change the server code,  the server has to do on sys.maxsize always your new value. 

We likly have to deprecate the API that the client sends the days at all. 

Because the user can't alter the value we should force the value on the server, or/and remove the form data of days

please fix it in the server code
https://github.com/Open-MSS/MSS/blob/develop/mslib/mscolab/server.py#L678

And on a migration script for the database we can adjust that number too.
Please add a ToDo or issue as follow up.

Reply:
I do not understand the migration scripts in /home/icg173/src/mss/mslib/mscolab/migrations/versions in particular the hashes? What do they relate to. I do not find them in git log.

Reply:
The server checks the value supplied by client and truncates the value. So old clients should work as well. "Normal" legit use for days>99999 are not possible.

Reply:
> I do not understand the migration scripts in mss/mslib/mscolab/migrations/versions in particular the hashes? What do they relate to. I do not find them in git log. I am also not sure that a fix is necessary as the server will crash _before_ setting an illegal date, so the dates in the database should be fine. So this can probably be merged as is.

In the database the hashes are set and connected by that to the scripts. 


https://www.digitalocean.com/community/tutorials/how-to-perform-flask-sqlalchemy-migrations-using-flask-migrate